lib/rbac/seed.rb

```diff
@@ -123,7 +123,7 @@ def find_uuid(type, data, name)
       result.uuid
     end
 
-    def create_request(user_file)
+    def make_request(user_file = nil)
```

Comment:
@syncrou we always need the user to pass in a file name, we can't have nil as the default.